src/language-graphql/printer-graphql.js

```diff
@@ -126,9 +126,13 @@ function genericPrint(path, options, print) {
     }
     case "StringValue": {
       if (n.block) {
-        // It is not possible to distinguish between """\nabc\n""" and """abc"""
-        // https://github.com/graphql/graphql-js/issues/1188
-        return options.originalText.slice(util.locStart(n), util.locEnd(n));
+        return concat([
+          '"""',
+          hardline,
+          n.value.replace(/"""/g, "\\$&"),
+          hardline,
+          '"""'
+        ]);
```

Comment:
Don’t we need to split on newlines and `concat` them, so that all lines get the correct base indentation?

I’d like to see this test:

```graphql
type Foo {
"""
This is a description
of the `one` field.
"""
  one: Type
}
```

turn into:

```graphql
type Foo {
  """
  This is a description
  of the `one` field.
  """
  one: Type
}
```

However, when I tried that in the [PR playground](https://deploy-preview-3632--prettier.netlify.com/playground/#N4Igxg9gdgLgprEAuEMCeAHOACAYhCbYAHSmJHMqgBUALASwGdsnsBDbAEzkbACd6GGPWikIAM2wxaOAAbQ4s7OPpwANpwB0pShQpRs2BUmzVMcUgF9SIADQgIQkVEbJQbPnwgB3AAoeEVxQ2ADcIek47EDZGGGRxNjVGOHsAIz42MABrOBgAZQxM+igAc2QYPgBXFJBOCDB4xOT7YuS+GF8MkoBbNkakmoArRgAPACEM7Ny8tm64ABliuH7mh0qYDHWAJhWawr425BASjIxaAEc1KIwBWAB1COlkAA4ABnsbiGS706ObnjgfBCy3sfDg50q9DBnTYPT6SASA3syW69HKVRqjGKJTUcAAipUIPBdvYYGxUg9OE8kFtSRl6GpsQBhCDdXpHKAKKKVZLUclBRHJSyWIA) that doc string wasn’t printed at all...

Reply:
Turns out i'm not printing the description for "FieldDefinition". We should look at all the places where descriptions are valid and make sure that they are all correctly printed.